bevy_rapier2d/examples/testbed2.rs

```diff
@@ -224,7 +224,7 @@ fn main() {
         .add_systems(
             Update,
             (
-                ui_example_system,
+                //ui_example_system,
```

Comment:
Is this waiting for some dependency upgrade or is it no longer needed?

Reply:
Thanks! I uncommented thoses :+1: